test-case/check-reboot.sh

```diff
@@ -0,0 +1,130 @@
+#!/bin/bash
+
+##
+## Case Name: test reboot
+## Preconditions:
+##    N/A
+## Description:
+##    run reboot for the test
+## Case step:
+##    1. check system status is correct
+##    2. wait for the delay time
+##    3. trigger for the reboot
+## Expect result:
+##    Test execute without report error in the LOG
+##
+
+source $(dirname ${BASH_SOURCE[0]})/../case-lib/lib.sh
+
+OPT_OPT_lst['l']='loop'     OPT_DESC_lst['l']='loop count'
+OPT_PARM_lst['l']=1         OPT_VALUE_lst['l']=3
+
+OPT_OPT_lst['d']='delay'    OPT_DESC_lst['d']='delay after system bootup'
+OPT_PARM_lst['d']=1         OPT_VALUE_lst['d']=2
+
+func_opt_parse_option $*
+
+func_lib_check_sudo
+loop_count=${OPT_VALUE_lst['l']}
+delay=${OPT_VALUE_lst['d']}
+
+# write the total & current count to the status file
+status_log=$LOG_ROOT/status.txt
+echo $loop_count >> $status_log
+count=$(head -n 1 $status_log|awk '{print $1;}')
+current=$[ $count - $loop_count ]
+
+dlogi "Round: $current/$count"
+
+# verify-pcm-list.sh & verify-tplg-binary.sh need TPLG file
+export TPLG=$(sof-get-default-tplg.sh)
+
+declare -a verify_lst
```

Comment:
Here you use a list to note which case to run after reboot, is it better to use some configure file? Then if we want to add one more case to run after reboot, we only need to add an item in configuration file.

Reply:
1. add configure file means all same type case need the configure file
2. mapping it as parameter need to handle too many option
3. if load the complex test-case, I think the case description also need to update

So I more like to keep this list/use the scan to run all `verify-*` script?

Reply:
for keep the list / run all `verify-*`, I agree on keep the list, this will limit the influence within the script, the latter will add a rule for script naming, I think that's not good. @xiulipan

Reply:
@Bin-QA @aiChaoSONG Configurable list would be better.
But how could we configure this list? by parameters or by files.